modules/ROOT/pages/setup.adoc

```diff
@@ -35,17 +35,17 @@ If you are unsure about the Mule runtime capacity required to handle your B2B tr
 == Environment Configuration
 
 You'll need to set up and configure your environment before you begin using Partner Manager.
```

Comment:
```suggestion
Before you begin using Anypoint Partner Manager in Anypoint Platform, you'll need to set up and configure your environment with Partner Manager roles, associated Mule deployment instances, and a payload storage API.
```

Reply:
REMEMBER: the texts that introduce each topic set must summarize the subtopics within, to prepare the audience for the content of the set. I am providing these now, but in future will comment only to have you do it.

Reply:
Okay, thanks.